code/modules/mob/living/carbon/human/species/slime.dm

```diff
@@ -74,6 +74,10 @@
 	recolor = new()
 	recolor.Grant(H)
 	ADD_TRAIT(H, TRAIT_WATERBREATH, "species")
+	RegisterSignal(H, COMSIG_HUMAN_UPDATE_DNA, /datum/species/slime/./proc/blend)
+	if(H.blood_color == null)
```

Comment:
Can be removed since you'd also want to change the `blood_color` whenever a human turns into a slime I recon. Changing his blood from red to ... well whatever color he is then